src/LandingPage.css

```diff
@@ -28,16 +28,84 @@ h1, h3 {
     padding-left: 35px;
 }
 
+.round-input{
+    background-color:rgb(196, 197, 201);
+    border:2px solid;
+    color: #6D6E71;
+    padding: 20px;
+    text-align:center;
+    text-decoration:none;
+    display:inline;
+    font-size: 16px;
+    margin: 4px 2px;
+    height:60px;
+    border-radius: 8px;
+}
+
+.centered{
+    margin-left: auto;
+    margin-right:auto;
+    text-align:center;
+    width:80%;
+}
+
+.fake-button-green{
```

Comment:
Use bootstrap button styling

Reply:
I didn't realize we could use bootstrap. That makes things way easier. I'll make the changes.